app.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+#
+# This script is run by OpenShift's s2i. Here we guarantee that we run desired
+# command
+#
+
+set -o nounset
+set -o errexit
+set -o errtrace
+set -o pipefail
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+
+THOTH_GRAPH_METRICS_EXPORTER=${THOTH_GRAPH_METRICS_EXPORTER:?'THOTH_GRAPH_METRICS_EXPORTER is not selected!'}
+
+if [ "$THOTH_GRAPH_METRICS_EXPORTER" = "graph_corruption_check" ]; then
+    exec python3 graph_corruption_check.py
```

Comment:
I don't see any need to have this file present. You can remove it and rename `graph_corruption_check.py` to `app.py`.

Reply:
But I thought we could run different task for the different long running query, like a cronworkflow with three tasks:

amcheck, table bloat data and index bloat data

Reply:
It still does not require this complex project structure. You can keep all things in the `app.py` and behave based on the parameters supplied. It simplifies project structure, maintenance, and testing the application.

Reply:
I ll use one then :) Thanks!!